deploy/scripts/src/mobile/utils/test-client.ts

```diff
@@ -19,5 +19,5 @@ export function parseTestClientResponse (response: Response, location: 'WebLocat
     throw new Error('Failed to parse URL from response')
   }
 
-  return url
+  return url.replace(config.backendUrl, `${config.backendUrl}/`)
```

Comment:
Another potential option which might be preferred over using replace would be to look at what we do in the TESTCLIENT_START test script in the Postman collection? The rationale for this approach would be that if we were to fix the test client bug, we wouldn't be required to make changes to the performance test repo. 

In the Postman test script, we parse the ApiLocation to extract the query parameters, and then append these to `<backendUrl>/verifyAuthorizeRequest?`. Are you able to have a look into doing it this way? Happy to talk through with you if any of that is confusing!

Reply:
In the Postman collection, the request_jwt and client_id were added as environment variables, not from the test client response. 
I think we should update the bug so we'll know that we should remove the replace from this method when fixed instead of adding a different implementation as a workaround for a bug 🤔

Reply:
The request_jwt and client_id are set here, after we make a request to the test client: 
![image](https://github.com/alphagov/di-devplatform-performance/assets/109979772/f32bb7ab-fadc-46d9-8ac1-27507bbcc33b)

We do have default values in the env vars, but we override them with the test client response's ApiLocation